docs/configuration/validation/brk.rst

```diff
@@ -0,0 +1,43 @@
+.. _configuration_validation_brk:
+
+=========================
+BRK - Zakelijk gerechtigd
+=========================
+
+The special *AddressNL* component has a specific validator available, that can
+check if the currently logged in user is the business owner of the filled-in property.
+
+Configuration
+=============
+
+1. Obtain a BRK API key.
+2. In Open Forms, navigate to: **Miscellaneous** > **KvK configuration**.
+3. Click **Add** next to the **KvK API Basisprofiel** field and fill in the following
+   details:
+
+   * **Label**: *Fill in a human readable label*, for example: ``My BRK API``
```

Comment:
```suggestion
   * **Label**: *Fill in a human readable label*, for example: ``Basisregistratie Kadaster (BRK)``
```
it's a central service - see https://www.kadaster.nl/zakelijk/registraties/basisregistraties/brk

What *can* happen is that the organisation itself requires access to go through their own gateway, so a different host can be configured.